modules/apps/commerce/commerce-product-test/src/testFunctional/macros/CommerceChannels.macro

```diff
@@ -76,6 +76,56 @@ definition {
 		Button.clickSave();
 	}
 
+	macro addFixedTaxRate {
+		var fieldLabel = "Amount";
+
+		Click(
+			key_entryName = "Fixed Tax Rate",
+			locator1 = "CommerceEntry#CHANNELS_GENERAL_TAB_EDITS");
+
+		SelectFrame.selectFrameNoLoading(locator1 = "CommerceEntry#IFRAME_SIDE_PANEL");
+
+		Check.checkToggleSwitch(
+			key_toggleSwitchLabel = "Active",
+			locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+
+		Button.clickSave();
+
+		if (isSet(percentage)) {
+			Check.checkToggleSwitch(
+				key_toggleSwitchLabel = "Percentage",
+				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+
+			Button.clickSave();
+
+			var fieldLabel = "Rate";
+		}
+
+		CommerceEntry.gotoMenuTab(menuTab = "Tax Rates");
+
+		Click(locator1 = "Button#PLUS");
+
+		SelectFrameTop();
+
+		SelectFrame(locator1 = "IFrame#MODAL_BODY");
+
+		Select(
+			key_fieldLabel = "Tax Category",
+			locator1 = "Select#GENERIC_SELECT_FIELD",
+			value1 = ${taxCategory});
+
+		Type(
+			key_fieldLabel = ${fieldLabel},
+			locator1 = "TextInput#GENERIC_TEXT_INPUT",
+			value1 = ${quantity});
+
+		Button.clickSubmitButton();
+
+		Refresh();
+
+		Button.clickSave();
```

Comment:
This can be a separate macro that is called by the parent macro, see comment above